datafusion/proto/src/lib.rs

```diff
@@ -367,15 +374,20 @@ mod roundtrip_tests {
         ];
 
         for test_case in should_fail_on_seralize.into_iter() {
-            let res: std::result::Result<
-                super::protobuf::ScalarValue,
-                super::to_proto::Error,
-            > = (&test_case).try_into();
-            assert!(
-                res.is_err(),
-                "The value {:?} should not have been able to serialize. Serialized to :{:?}",
-                test_case, res
-            );
+            let proto: Result<super::protobuf::ScalarValue, super::to_proto::Error> =
```

Comment:
Some of the validation is now done during deserialization rather than serialization so I had to change this test slightly